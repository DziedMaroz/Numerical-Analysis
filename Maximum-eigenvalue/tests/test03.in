4
1	2	3	10
3	2	4	100
7	8	90	31
34	32	4	15
1	2	3	5
0.000003
