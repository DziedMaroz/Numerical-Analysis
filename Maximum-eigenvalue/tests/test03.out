96.61095118459914
